tools/update-eslint.sh

```diff
@@ -19,7 +19,7 @@ rm -rf node_modules/eslint
     (
         cd node_modules/eslint || exit
 
-        npm install --no-bin-links --production --no-package-lock eslint-plugin-markdown@latest
+        npm install --no-bin-links --production --no-package-lock git+https://github.com/eslint/eslint-plugin-markdown.git#pull/172
```

Comment:
This is not ideal, we probably need a longer term solution: https://github.com/orgs/nodejs/teams/linting/discussions/1